detect.go

```diff
@@ -0,0 +1,44 @@
+package condaenvupdate
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/paketo-buildpacks/packit"
+)
+
+// Detect returns a packit.DetectFunc that will be invoked during the
+// detect phase of the buildpack lifecycle.
+//
+// Detection passes when there is an environment.yml file in the app directory,
+// and will contribute a Build Plan that provides conda-environment and requires
+// conda.
+func Detect() packit.DetectFunc {
+	return func(context packit.DetectContext) (packit.DetectResult, error) {
+		_, err := os.Stat(filepath.Join(context.WorkingDir, "environment.yml"))
```

Comment:
According to the integration testdata, you don't even need environment.yml for vendored apps. If that's the case - can we detect if either of this file or package-list.txt is found?
It seems a little not nice to ask users to create an environment.yml just to pass buildpack detection?